CMakeLists.txt

```diff
@@ -438,10 +446,10 @@ set(open62541_LIBRARIES "")
 set(open62541_PUBLIC_LIBRARIES "")
 if("${UA_ARCHITECTURE}" STREQUAL "posix")
     list(APPEND open62541_LIBRARIES "m")
-    if(UA_MULTITHREADING GREATER_EQUAL 100 OR UA_BUILD_UNIT_TESTS)
+    if(UA_MULTITHREADING GREATER_EQUAL 100 OR UA_BUILD_UNIT_TESTS AND NOT ANDROID_NDK_TOOLCHAIN_INCLUDED)
```

Comment:
I personally (not open62541 maintainer) prefer parentheses enforcing precedence when AND and OR is compounded like this, so I don't have to remember what language I'm reading and what are the precedence rules for this language.

Reply:
thank you for the document. I prefer to use parenthesis when there are lots of calculations but if it is necessary for cmake or open5241, I can change it.